DOCKER/README.md

```diff
@@ -12,28 +12,25 @@
 - `0.9.1`, `0.9`, [(Dockerfile)](https://github.com/tendermint/tendermint/blob/809e0e8c5933604ba8b2d096803ada7c5ec4dfd3/DOCKER/Dockerfile)
 - `0.9.0` [(Dockerfile)](https://github.com/tendermint/tendermint/blob/d474baeeea6c22b289e7402449572f7c89ee21da/DOCKER/Dockerfile)
 - `0.8.0`, `0.8` [(Dockerfile)](https://github.com/tendermint/tendermint/blob/bf64dd21fdb193e54d8addaaaa2ecf7ac371de8c/DOCKER/Dockerfile)
-- `develop` [(Dockerfile)](https://github.com/tendermint/tendermint/blob/master/DOCKER/Dockerfile.develop)
-
-`develop` tag points to the [develop](https://github.com/tendermint/tendermint/tree/develop) branch.
 
 ## Quick reference
 
-* **Where to get help:**
-  https://cosmos.network/community
+- **Where to get help:**
+  [cosmos.network/ecosystem](https://cosmos.network/ecosystem)
 
-* **Where to file issues:**
-  https://github.com/tendermint/tendermint/issues
+- **Where to file issues:**
+  [Tendermint Issues](https://github.com/tendermint/tendermint/issues)
 
-* **Supported Docker versions:**
+- **Supported Docker versions:**
   [the latest release](https://github.com/moby/moby/releases) (down to 1.6 on a best-effort basis)
 
 ## Tendermint
 
 Tendermint Core is Byzantine Fault Tolerant (BFT) middleware that takes a state transition machine, written in any programming language, and securely replicates it on many machines.
 
-For more background, see the [introduction](https://tendermint.readthedocs.io/en/master/introduction.html).
+For more background, see the [the docs](https://tendermint.com/docs/introduction/#quick-start).
```

Comment:
Just asking, the `en` is no more supported or was the only available language?

Reply:
copied the link from online, doesn't seem the multiple langs are supported anymore